app/screens/index.js

```diff
@@ -62,43 +21,43 @@ function wrapWithContextProvider(Comp, excludeEvents = true) {
-    Navigation.registerComponent('EditPost', () => wrapWithContextProvider(EditPost), store, Provider);
-    Navigation.registerComponent('EditProfile', () => wrapWithContextProvider(EditProfile), store, Provider);
-    Navigation.registerComponent('ImagePreview', () => wrapWithContextProvider(ImagePreview), store, Provider);
-    Navigation.registerComponent('Login', () => wrapWithContextProvider(Login), store, Provider);
-    Navigation.registerComponent('LoginOptions', () => wrapWithContextProvider(LoginOptions), store, Provider);
-    Navigation.registerComponent('MFA', () => wrapWithContextProvider(Mfa), store, Provider);
-    Navigation.registerComponent('MoreChannels', () => wrapWithContextProvider(MoreChannels), store, Provider);
-    Navigation.registerComponent('MoreDirectMessages', () => wrapWithContextProvider(MoreDirectMessages), store, Provider);
-    Navigation.registerComponent('Notification', () => wrapWithContextProvider(Notification), store, Provider);
-    Navigation.registerComponent('NotificationSettings', () => wrapWithContextProvider(NotificationSettings), store, Provider);
-    Navigation.registerComponent('NotificationSettingsEmail', () => wrapWithContextProvider(NotificationSettingsEmail), store, Provider);
-    Navigation.registerComponent('NotificationSettingsMentions', () => wrapWithContextProvider(NotificationSettingsMentions), store, Provider);
-    Navigation.registerComponent('NotificationSettingsMentionsKeywords', () => wrapWithContextProvider(NotificationSettingsMentionsKeywords), store, Provider);
-    Navigation.registerComponent('NotificationSettingsMobile', () => wrapWithContextProvider(NotificationSettingsMobile), store, Provider);
-    Navigation.registerComponent('OptionsModal', () => wrapWithContextProvider(OptionsModal), store, Provider);
-    Navigation.registerComponent('Root', () => Root, store, Provider);
-    Navigation.registerComponent('Search', () => wrapWithContextProvider(Search), store, Provider);
-    Navigation.registerComponent('SelectServer', () => wrapWithContextProvider(SelectServer), store, Provider);
-    Navigation.registerComponent('SelectTeam', () => wrapWithContextProvider(SelectTeam), store, Provider);
-    Navigation.registerComponent('Settings', () => wrapWithContextProvider(Settings), store, Provider);
-    Navigation.registerComponent('SSO', () => wrapWithContextProvider(SSO), store, Provider);
-    Navigation.registerComponent('Table', () => wrapWithContextProvider(Table), store, Provider);
-    Navigation.registerComponent('TableImage', () => wrapWithContextProvider(TableImage), store, Provider);
-    Navigation.registerComponent('Thread', () => wrapWithContextProvider(Thread), store, Provider);
-    Navigation.registerComponent('UserProfile', () => wrapWithContextProvider(UserProfile), store, Provider);
+    Navigation.registerComponent('About', () => wrapWithContextProvider(require('app/screens/about').default), store, Provider); //eslint-disable-line global-require
```

Comment:
You could do this here
```js
export function registerScreens(store, Provider) {
    /* eslint-disable global-require */
   ...
}
```

that way you do not need to add the `eslint-disable-line` in each line